Ginger/GingerCore/ALM/Qtest/ImportFromQtest.cs

```diff
@@ -0,0 +1,1128 @@
+
+                        //get the param value
+                        if (tcParameter != null && tcParameter.Value != null && tcParameter.Value != string.Empty)
+                            paramSelectedValue = tcParameter.Value;
+                        else
+                        {
+                            isflowControlParam = null;//empty value
+                            paramSelectedValue = "<Empty>";
+                        }
+
+                        //check if parameter is part of a link
+                        string linkedVariable = null;
+                        if (paramSelectedValue.StartsWith("#$#"))
+                        {
+                            string[] valueParts = paramSelectedValue.Split(new string[] { "#$#" }, StringSplitOptions.None);
+                            if (valueParts.Count() == 3)
+                            {
+                                linkedVariable = valueParts[1];
+                                paramSelectedValue = "$$_" + valueParts[2];//so it still will be considered as non-flow control
+
+                                if (busVariables.Keys.Contains(linkedVariable) == false)
+                                {
+                                    busVariables.Add(linkedVariable, valueParts[2]);
+                                }
+                            }
+                        }
+
+                        //determine if the param is Flow Control Param or not based on it value and agreed sign "$$_"
+                        if (paramSelectedValue.StartsWith("$$_"))
+                        {
+                            isflowControlParam = false;
+                            if (paramSelectedValue.StartsWith("$$_"))
+                                paramSelectedValue = paramSelectedValue.Substring(3);//get value without "$$_"
+                        }
+                        else if (paramSelectedValue != "<Empty>")
+                            isflowControlParam = true;
+
+                        //check if already exist param with that name
+                        VariableBase stepActivityVar = stepActivity.Variables.Where(x => x.Name.ToUpper() == param.ToUpper()).FirstOrDefault();
+                        if (stepActivityVar == null)
+                        {
+                            //#Param not exist so add it
+                            if (isflowControlParam == true)
+                            {
+                                //add it as selection list param                               
+                                stepActivityVar = new VariableSelectionList();
+                                stepActivityVar.Name = param;
+                                stepActivity.AddVariable(stepActivityVar);
+                                stepActivity.AutomationStatus = eActivityAutomationStatus.Development;//reset status because new flow control param was added
+                            }
+                            else
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)